src/world/common/FoliageTransform.inc.c

```diff
@@ -0,0 +1,80 @@
+#include "common.h"
```

Comment:
This file is in src/**world**/common/, which is a bit weird considering this is used in battles too. Maybe `src/common` is better? @ethteck may have opinions